dotcom-rendering/src/web/layouts/FrontLayout.tsx

```diff
@@ -48,6 +50,21 @@ export const FrontLayout = ({ front, NAV }: Props) => {
 		<>
 			<div data-print-layout="hide" id="bannerandheader">
 				<>
+					<Stuck>
+						<ElementContainer
+							showTopBorder={false}
+							showSideBorders={false}
+							padded={false}
+							shouldCenter={false}
+						>
+							<HeaderAdSlot
+								isAdFreeUser={front.isAdFreeUser}
+								shouldHideAds={false}
```

Comment:
Not entirely sure what this prop does, but it sounds like it shouldn't always be false? 
```
shouldHideAds={front.isAdFreeUser}
```

Reply:
Good question.

`shouldHideAds` is a property that gets set on an article if the content is sensitive. It's false here because there's no equivalent for fronts

Reply:
My next PR will refactor these props to make this read better